fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/NavigationBar.kt

```diff
@@ -36,12 +37,14 @@ import org.mozilla.fenix.theme.Theme
  * Top-level UI for displaying the navigation bar.
  *
  * @param actionItems A list of [ActionItem] used to populate the bar.
- * @param menuButton A [MenuButton] to be used for [ItemType.MENU]
+ * @param menuButton A [MenuButton] to be used for [ItemType.MENU].
+ * @param tabCount The number of opened tabs.
  */
 @Composable
 fun NavigationBar(
     actionItems: List<ActionItem>,
     menuButton: MenuButton? = null,
+    tabCount: Int,
```

Comment:
We want to put the required parameters before the optional parameters. For the most part, you will see we follow https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-component-api-guidelines.md#parameters-order. The except we implement is that lambdas can appear below optional parameters.